index.js

```diff
@@ -3,6 +3,8 @@
 	Author Tobias Koppers @sokra
 */
 
+const path = require('path');
```

Comment:
I just noticed that `var` is used everywhere else in this file. It might be worth staying consistent here.